.github/workflows/developments_datasync.yml

```diff
@@ -67,19 +62,31 @@ jobs:
       BUILD_ENGINE: postgresql://postgres:postgres@localhost:5432/postgres
     steps:
       - uses: actions/checkout@v3
+
+      - name: install dependencies ...
+        working-directory: db-developments
+        run: |
+          sudo apt update
+          curl -O https://dl.min.io/client/mc/release/linux-amd64/mc
+          chmod +x mc
+          sudo mv ./mc /usr/bin
+          mc alias set spaces $AWS_S3_ENDPOINT $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY --api S3v4
+
       - name: Load to Database (workflow_dispatch)
         if: github.event_name == 'workflow_dispatch'
         working-directory: db-developments
```

Comment:
Just fyi, you can set the default working dir or shell at the job level, e.g. [here](https://github.com/NYCPlanning/data-engineering/pull/41/files#diff-b2d695d3b864a94b1de11ae76cbf468bab8f2e856db07aff558306a3dbe58f7aR21)